virttest/utils_test/libvirt.py

```diff
@@ -1585,10 +1585,10 @@ def check_exit_status(result, expect_error=False):
     """
     if not expect_error:
         if result.exit_status != 0:
-            raise exceptions.TestFail(results_stderr_52lts(result))
+            raise exceptions.TestFail(result.stderr_text)
         else:
             logging.debug("Command output:\n%s",
-                          results_stdout_52lts(result).strip())
+                          results.stdout_text.strip())
```

Comment:
change results_stdout_text to result.stdout_text

Reply:
Yep, thanks!